### PR TITLE
BETA: Register aymouwu.is-a.dev

### PR DESCRIPTION
**Pull request by NotAymo**
Added `aymouwu.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by NotAymo** [comment]
please remove the other domains, let only aymouwu.is-a.dev, thanks in advance

File: domains/cdn.untendo.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "NotAymo",
+        "email": "ninecraftoff@outlook.com"
+    },
+    "record": {
+        "CNAME": "untendo.github.io"
```
Comment (Stef-00012) [inline domains/cdn.untendo.json:7]: I cannot reach your website (404 page)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/cdn.untendo.json (hunk 1 above)
Comment (NotAymo) [reply domains/cdn.untendo.json]: done

**Comment by Stef-00012** [comment]
seems good now


> please remove the other domains, let only aymouwu.is-a.dev, thanks in advance

I didn't even notice this comment 💀

**Review by iostpa — APPROVED** [review]
seems good to me

**Pull request merged**